gradio/_simple_templates/simpletextbox.py

```diff
@@ -0,0 +1,100 @@
+"""gr.Textbox() component."""
+
+from __future__ import annotations
+
+from typing import Any, Callable
+
+from gradio.components.base import FormComponent
+from gradio.events import Events
+
+
+class SimpleTextbox(FormComponent):
+    """
+    Creates a textarea for user to enter string input or display string output.
+    Preprocessing: passes textarea value as a {str} into the function.
+    Postprocessing: expects a {str} returned from function and sets textarea value to it.
+    Examples-format: a {str} representing the textbox input.
+    """
+
+    EVENTS = [
+        Events.change,
+        Events.input,
+        Events.select,
+        Events.submit,
+        Events.focus,
+        Events.blur,
+    ]
+
+    def __init__(
+        self,
+        value: str | Callable | None = "",
+        *,
+        placeholder: str | None = None,
+        label: str | None = None,
+        every: float | None = None,
+        show_label: bool | None = None,
+        scale: int | None = None,
+        min_width: int = 160,
+        interactive: bool | None = None,
+        visible: bool = True,
+        rtl: bool = False,
+        elem_id: str | None = None,
+        elem_classes: list[str] | str | None = None,
```

Comment:
I guess these are all pretty standard for all components (or essential)

Reply:
Yea the only ones that are "textbox-specific" here are `rtl` and `placeholder`. All the other ones are pretty standard for it to look like the other gradio components.